aws/sdk/integration-tests/kms/tests/sensitive-it.rs

```diff
@@ -54,6 +54,13 @@ fn client_is_clone() {
     let _ = client.clone();
 }
 
+#[test]
```

Comment:
Can you use the pattern above to make this a compile failure instead of a test failure?

Reply:
Sure, I'll copy the Send + Sync test.